src/domain/occurrence/enrolmentDetails/__tests__/EnrolmentDetails.test.tsx

```diff
@@ -75,16 +109,14 @@ test('renders correct information', async () => {
 
 test('enrolment action buttons work correctly', async () => {
   const { container } = render(
-    <MemoryRouter>
-      <MockedProvider mocks={mocks}>
-        <EnrolmentDetails
-          enrolmentId="RW5yb2xtZW50Tm9kZTo1Ng=="
-          eventId=""
-          occurrenceId=""
-          onGoBackClick={jest.fn()}
-        />
-      </MockedProvider>
-    </MemoryRouter>
+    <EnrolmentDetails
+      enrolmentId="RW5yb2xtZW50Tm9kZTo1Ng=="
```

Comment:
Maybe a constant for id?

Reply:
Maybe event helper function to render the component. 
```
const { container } = render(
    <EnrolmentDetails
      enrolmentId="RW5yb2xtZW50Tm9kZTo1Ng=="
      eventId=""
      occurrenceId=""
      onGoBackClick={jest.fn()}
      refetchOccurrence={jest.fn()}
    />,
    { mocks }
  );
```
is repeated 3 times

Reply:
improved